src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -0,0 +1,200 @@
+    body={
+      <HovercardBody>
+        {commits !== undefined && (
+          <React.Fragment>
+            <div className="divider">
+              <h6>{t('Commits')}</h6>
+            </div>
+            <ReasonGroup>
+              {commits.slice(0, 6).map(({message, dateCreated}, i) => (
+                <ReasonItem key={i}>
+                  <CommitIcon />
+                  <CommitMessage message={message} date={dateCreated} />
+                </ReasonItem>
+              ))}
+            </ReasonGroup>
+          </React.Fragment>
+        )}
+        {rules !== undefined && (
+          <React.Fragment>
+            <div className="divider">
+              <h6>{t('Ownership Rules')}</h6>
+            </div>
+            <ReasonGroup>
+              {rules.map(([type, matched], i) => (
+                <ReasonItem key={i}>
+                  <OwnershipTag tagType={type} />
+                  <OwnershipValue>{matched}</OwnershipValue>
+                </ReasonItem>
+              ))}
+            </ReasonGroup>
+          </React.Fragment>
+        )}
+      </HovercardBody>
+    }
+    {...props}
+  />
+);
+
+SuggestedOwnerHovercard.propTypes = {
+  /**
+   * The suggested actor.
+   */
+  actor: PropTypes.oneOfType([
+    // eslint-disable-next-line react/no-typos
+    SentryTypes.User,
+    // Sentry user which has *not* been expanded
+    PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+    // Unidentifier user (from commits)
+    PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+    // Sentry team
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+  ]),
+
+  /**
+   * The list of commits the actor is suggested for. May be left blank if the
+   * actor is not suggested for commits.
+   */
+  commits: PropTypes.arrayOf(
+    PropTypes.shape({
+      message: PropTypes.string.isRequired,
+      dateCreated: PropTypes.string.isRequired,
+    })
+  ),
+
+  /**
+   * The list of ownership rules the actor is suggested for. Maybe left blank
+   * if the actor is not suggested based on ownership rules.
+   */
+  rules: PropTypes.arrayOf(PropTypes.array),
+};
+
+const tagColors = {
+  url: 'greenLight',
+  path: 'blueLight',
+};
+
+const CommitIcon = styled(p => <InlineSvg src="icon-commit" size="16px" {...p} />)`
+  margin-right: 4px;
+  flex-shrink: 0;
+`;
+
+const CommitMessage = styled(({message, date, ...props}) => (
+  <div {...props}>
+    {message.split('\n')[0]}
+    <CommitDate date={date} />
+  </div>
+))`
+  color: ${p => p.theme.gray5};
+  font-size: 11px;
+  margin-top: 2px;
+`;
+
+const CommitDate = styled(({date, ...props}) => (
+  <div {...props}>{moment(date).fromNow()}</div>
+))`
+  margin-top: 4px;
+  color: ${p => p.theme.gray2};
+`;
+
+const ReasonGroup = styled('ul')`
+  list-style: none;
```

Comment:
do we need this to be a `<ul>`? if not just switch to a `div` and delete this block.

Reply:
Our `li` would just become a div as well in that case. I did this for semantics since it is a list of things.

Reply:
Ok. Ultimately it's up to you. Unless a semantic element provides some kind of usability or accessibility benefit I'm inclined to choose the route that favors less code. But again I don't think it's a big deal and you certainly aren't the first person to do it.

Reply:
Here is a nice lil' write up on why `<li>` elements potentially don't do anything (from my alma-mater, to boot!): 

https://css-tricks.com/navigation-in-lists-to-be-or-not-to-be/